src/NuGet.Clients/NuGet.PackageManagement.UI/Actions/UIActionEngine.cs

```diff
@@ -561,6 +562,25 @@ private static string GenerateUpgradeReport(string projectName, string backupPat
                 }
             }, cancellationToken);
         }
+        internal static TelemetryEvent ToTelemetryPackage(string packageId, string packageVersion, string packageVersionRange)
```

Comment:
format: You're missing an extra line. Should have been flagged by the analyzer.

Reply:
Fixed!